cmake/lsan.suppressions

```diff
@@ -0,0 +1,2 @@
+# LSAN suppression for ltdl library known issue.
```

Comment:
I think that this file is not related to CMake. Can we consider different locations, e.g. `<root>/sanitizers/`? @AlexVeprev what do you think about this?